pvlib/pvsystem.py

```diff
@@ -157,13 +157,16 @@ class PVSystem:
         North=0, East=90, South=180, West=270.
 
     albedo : None or float, default None
-        The ground albedo. If ``None``, will attempt to use
-        ``surface_type`` and ``irradiance.SURFACE_ALBEDOS``
-        to lookup albedo.
+        Ground surface albedo. If ``None``, then ``surface_type`` is used
+        to look up a value in ``irradiance.SURFACE_ALBEDOS``.
+        If ``surface_type`` is also None then a ground surface albedo
+        of 0.25 is used. For time-dependent albedos, add ``'albedo'`` to
+        the input ``'weather'`` DataFrame for
+        :py:class:`pvlib.modelchain.ModelChain` methods.
```

Comment:
Worth mentioning that time-dependent albedos can be passed to `PVSystem.get_irradiance` too?

Reply:
I think less is more here, since `PVSystem.albedo` is deprecated